_includes/sections/voice-video-messenger.html

```diff
@@ -51,20 +19,30 @@ <h1 id="voip" class="anchor"><a href="#voip"><i class="fas fa-link anchor-icon">
 web=""
 %}
 
-
-
+{% include cardv2.html
+title="Mumble"
+image="/assets/img/tools/Mumble.png"
+website="https://mumble.info/"
+description="Mumble is an open source, low-latency, high quality voice chat software primarily intended for use while gaming. Note that while Mumble doesn't log messages or record by default, <a href=\"https://github.com/mumble-voip/mumble/issues/1813\">it's missing end-to-end encryption</a>, so self-hosting is recommended."
+forum="https://forum.privacytools.io/t/discussion-mumble/1289"
+github="https://github.com/mumble-voip/"
+windows=""
+linux=""
+mac=""
+%}
 
 <h3>Worth Mentioning</h3>
 
 <ul>
-  <li><a href="https://jitsi.org/">Jitsi</a> - Jitsi is a free and open source multiplatform voice (VoIP), videoconferencing and instant messaging application.</li>
-  <li> <a href="https://tox.chat/">Tox</a> - <span class="badge badge-warning">Experimental</span> A free and open-source, peer-to-peer, encrypted instant messaging and video calling software. </li>
-  <li><a href="https://jami.net/">Jami (formerly Ring/SFLphone)</a> - Gives you a full control over your communications and an unmatched level of privacy.</li>
+  <li><a href="https://jitsi.org/">Jitsi Meet</a> - Jitsi Meet is a free and open source multiplatform voice (VoIP), videoconferencing and instant messaging application.<span class=\"badge badge-warning\" data-toggle=\"tooltip\" title=\"Our Firefox tweaks recommend disabling WebRTC as it can be used to leak your IP address even behind a VPN, which is why Tor Browser disables it.\">Requires WebRTC</span> </li>
```

Comment:
@JonahAragon Did you remove the warning label?

Reply:
No, looking better the warning is there, but it's not visible on live site. I don't know when I can look at it better to fix it.

I will grant permissions to write in my fork in case it helps.

Reply:
I don't think I made any changes here. The issue is the backslashes, not sure where those are from.

```suggestion
  <li><a href="https://jitsi.org/">Jitsi Meet</a> - Jitsi Meet is a free and open source multiplatform voice (VoIP), videoconferencing and instant messaging application. <span class="badge badge-warning" data-toggle="tooltip" title="Our Firefox tweaks recommend disabling WebRTC as it can be used to leak your IP address even behind a VPN, which is why Tor Browser disables it.">Requires WebRTC</span></li>
```